src/dashboard/cd-dashboard-news.vue

```diff
@@ -1,14 +1,77 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-news">
-      <h3 class="cd-dashboard-news__header">Community News and Forum Updates</h3>
+      <h2 class="cd-dashboard-news__header">Community News and Forum Updates</h2>
+      <div class="cd-dashboard-news__posts" v-for="post in allPosts">
+        <span class="cd-dashboard-news__posts-left">
+          <p class="cd-dashboard-news__post-type">{{ post.type }}</p>
+          <p class="cd-dashboard-news__post-date">{{ post.date }}</p>
+        </span>
+        <span class="cd-dashboard-news__posts-right">
+          <h4 class="cd-dashboard-news__post-title">
+            <a class="cd-dashboard-news__post-title-link" :href="`${post.link}`">{{ post.title }}</a>
+          </h4>
+        </span>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
+  import NewsForumsService from './service';
+
   export default {
     name: 'cd-dashboard-news',
+    data() {
+      return {
+        news: null,
+        forums: null,
+      };
+    },
+    computed: {
+      allPosts() {
+        if (this.news && this.forums) {
+          const joinedPosts = ((this.news).map(post => ({
+            type: 'News',
+            date: (post.date.split('T')[0]).split('-').join(''),
+            link: post.link,
+            title: post.title.rendered,
+          })).concat((this.forums).map(post => ({
+            type: 'Forums',
+            date: (post.timestampISO.split('T')[0]).split('-').join(''),
+            link: `https://forums.coderdojo.com/topic/${post.slug}`,
+            title: post.title,
+          }))));
+
+          return this.sortPostsByDate(joinedPosts).splice(0, 6);
+        }
+        return null;
+      },
+    },
+    methods: {
+      async loadNews() {
+        const res = await NewsForumsService.loadNews();
+        this.news = res.body;
+      },
+      async loadForums() {
+        const res = await NewsForumsService.loadForums();
+        this.forums = res.body.topics;
+      },
+      sortPostsByDate(posts) {
+        const sortedPosts = posts.sort((a, b) =>
+         a.date - b.date);
+        return (sortedPosts.map(post => (Object.assign({
+          type: post.type,
+          date: [post.date.slice(0, 4), post.date.slice(4, 6), post.date.slice(6, 8)].reverse().join('/'),
+          link: post.link,
+          title: post.title,
+        })))).reverse();
```

Comment:
Given we're writing the comparator for sorting, we shouldn't need to reverse afterwards. Using `b.date - a.date` in the comparator would sort it how we want.

If we parse the timestamps to actual dates though you'll need to adjust the comparator a bit anyway. Thankfully dates can be compared with each other, so it's still pretty straight forward.